packages/calling-component-bindings/src/providers/CallAgentProvider.tsx

```diff
@@ -25,6 +36,16 @@ const CallAgentProviderBase = (props: CallAgentProviderProps): JSX.Element => {
   return <CallAgentContext.Provider value={initialState}>{props.children}</CallAgentContext.Provider>;
 };
 
+/**
+ * A {@link React.Context} that stores a {@link @azure/communication-calling#CallAgent}.
+ *
+ * Calling components from this package must be wrapped with a {@link CallAgentProvider}.
+ *
+ * @public
+ */
 export const CallAgentProvider = (props: CallAgentProviderProps): JSX.Element => <CallAgentProviderBase {...props} />;
 
+/**
```

Comment:
These should be exposed to enable people getting call/callAgent when they start with component level